src/05_lists.py

```diff
@@ -7,23 +7,35 @@
 # For the following, DO NOT USE AN ASSIGNMENT (=).
 
 # Change x so that it is [1, 2, 3, 4]
-# YOUR CODE HERE 
+# YOUR CODE HERE
+
+x.append(4)
 print(x)
 
 # Using y, change x so that it is [1, 2, 3, 4, 8, 9, 10]
-# YOUR CODE HERE 
+# YOUR CODE HERE
+
+x.extend(y)
 print(x)
 
 # Change x so that it is [1, 2, 3, 4, 9, 10]
-# YOUR CODE HERE 
+# YOUR CODE HERE
+
+x.remove(8)
 print(x)
 
 # Change x so that it is [1, 2, 3, 4, 9, 99, 10]
-# YOUR CODE HERE 
+# YOUR CODE HERE
+
+x.insert(-1, 99)
 print(x)
 
 # Print the length of list x
-# YOUR CODE HERE 
+# YOUR CODE HERE
+
+print(len(x))
 
 # Print all the values in x multiplied by 1000
-# YOUR CODE HERE
\ No newline at end of file
+# YOUR CODE HERE
+
+print([num * 1000 for num in x])
```

Comment:
I think creating a variable and printing it would make the code more readable. Like you did it in 01_bignum.py